marconi-chain-index/src/Marconi/ChainIndex/Logging.hs

```diff
@@ -54,6 +59,14 @@ marconiFormatting =
   Pretty.renderStrict
     . Pretty.layoutPretty Pretty.defaultLayoutOptions
 
+logMInfo :: MonadIO m => Pretty.Doc () -> ChainIndexerT m ()
+logMInfo msg = do
+  trace <- Lens.view runIndexerConfigTrace
+  lift $ logInfo trace msg
```

Comment:
You could make this into a helper function that you use to map the other log functions:

```suggestion
logToLogM :: MonadIO m => (MarconiTrace m -> Pretty.Doc () -> m ()) -> Pretty.Doc () -> ChainIndexerT m ()
logMInfo logf msg = do
  trace <- Lens.view runIndexerConfigTrace
  lift $ logf trace msg

logMInfo :: MonadIO m => Pretty.Doc () -> ChainIndexerT m ()
logMInfo = logToLogM logInfo
```

Reply:
Great idea, thanks!